cmd/ctr-remote/commands/rpull.go

```diff
@@ -94,7 +94,7 @@ func pull(ctx context.Context, client *containerd.Client, ref string, config *co
 		containerd.WithSchema1Conversion,
 		containerd.WithPullUnpack,
 		containerd.WithPullSnapshotter(remoteSnapshotterName),
-		containerd.WithImageHandlerWrapper(stargz.AppendInfoHandlerWrapper(ref)),
+		containerd.WithImageHandlerWrapper(stargz.AppendInfoHandlerWrapper(ref, 10000000)),
```

Comment:
`10000000` -> `10 * 1000 * 1000` (or `10 * 1024 * 1024`?) for readability

Reply:
Thanks for the comment. Fixed it.